### ISSUE TITLE
retry middleware: Cannot read properties of null 

### ISSUE THREAD

**Issue by dxptqhtlutehvlyxcmtg** [elbywan/wretch#145]
We're encountering a TypeError when a network error occurs while using the retry middleware with both `resolveWithLatestResponse` and `retryOnNetworkError` being true:

```typescript
retry({
    resolveWithLatestResponse: true,
    retryOnNetworkError: true,
})
```

When simulating a network error (by setting Chrome to offline in DevTools):

![Screenshot from 2022-09-27 13-10-06](https://user-images.githubusercontent.com/108966931/192627436-39b76cf7-88ea-4fe8-bd6d-e2fb17233a6c.png)

![Screenshot from 2022-09-27 13-11-05](https://user-images.githubusercontent.com/108966931/192627821-cc256a77-b594-4ecf-9593-25d2e02172d6.png)

This is on 2.1.2, and does not happen when either one of those options are false. Also, it's the only middleware in use.

**Comment by elbywan** [comment]
Hey @dxptqhtlutehvlyxcmtg,

> We're encountering a TypeError when a network error occurs while using the retry middleware with both resolveWithLatestResponse and retryOnNetworkError being true:

Thanks for reporting the issue 🙇.

I just published `2.1.3` which contains a fix, feel free to reopen if there is any remaining problem!

**Comment by dxptqhtlutehvlyxcmtg** [comment]
Awesome, thank you for the quick update!
